ext/mysqlnd/mysqlnd_ps.c

```diff
@@ -764,7 +764,7 @@ mysqlnd_fetch_stmt_row_cursor(MYSQLND_RES * result, zval **row_ptr, const unsign
 
 	DBG_ENTER("mysqlnd_fetch_stmt_row_cursor");
 
-	if (!stmt || !stmt->conn || !result || !result->conn || !result->unbuf) {
+	if (!stmt || !stmt->conn || !result->conn || !result->unbuf) {
```

Comment:
Is it possible that `result->unbuf`  is `null`?

Reply:
No, I missed this, I'll also remove this one.

Reply:
Also, since `stmt` is also dereferenced, it may be possible to delete it.